### PR TITLE
Contacts - Category Module [WIP]

### PR DESCRIPTION
**Pull request by brianteeman**
This module displays a list of contacts from one or more categories.

It is a fork of the similar article module
### Work In Progress

Admin is complete 
Front end still needs work (any help appreciated - related to https://github.com/joomla/joomla-cms/blob/a24839e8b0ab63c05d6e4c483278414631fd2be8/components/com_content/models/articles.php#L62)

Have not done the install yet so it will need to be discovered

### CODE REVIEW

**Comment by brianteeman** [comment]
I would really appreciate any help that can be given here.

File: components/com_content/helpers/query.php [hunk 1]
```diff
@@ -173,6 +173,11 @@ public static function getQueryDate($orderDate)
 				$queryDate = ' CASE WHEN a.publish_up = ' . $db->quote($db->getNullDate()) . ' THEN a.created ELSE a.publish_up END ';
 				break;
 
+			// Use created if publish_down is not set
+			case 'published' :
```
Comment (ggppdk) [inline components/com_content/helpers/query.php]: You meant 'unpublished'

File: components/com_content/views/category/tmpl/blog.xml [hunk 2]
```diff
@@ -279,6 +279,7 @@
 					<option value="created">JGLOBAL_CREATED</option>
 					<option value="modified">JGLOBAL_MODIFIED</option>
 					<option value="published">JPUBLISHED</option>
+					<option value="published">JUNPUBLISHED</option>
```
Comment (ggppdk) [inline components/com_content/views/category/tmpl/blog.xml]: You meant 'unpublished'

**Comment by brianteeman** [comment]
Closing for now.

Thanks for looking @ggppdk I have some files from another commit in here. I will start again on a clean branch

**Pull request closed without merging**